articles/protocols/saml/saml-configuration/troubleshoot/auth0-as-idp.md

```diff
@@ -0,0 +1,100 @@
+---
+description: How to troubleshoot SAML-related configuration issues when Auth0 is the identity provider
+toc: true
+---
+
+# SAML Troubleshooting When Auth0 is the Identity Provider
+
+When troubleshooting a SAML login, there are four primary stages to check:
+
+* Stage 1: The user is successfully redirected to IDP and is able to login.
+* Stage 2: After login with the IDP, the user returns to Auth0 with a successful login event recorded.
+* Stage 3: After a successful login event in Auth0, the user profile in Auth0 is correct.
+* Stage 4: The user successfully redirects back to application and is able to access application.
+
+The following sections describe how to check each stage and how to identify if there are any issues with a given stage.
+
+## Issue: User Successfully Logs In With the IdP, but a Successful Login Event Does *Not* Show Up in Auth0 Logs
```

Comment:
I would add something shorter for the title, like `Successful Login does not show up in Auth0 logs`. After that the first sentence of this paragraph could be the full one: `The user successfully logs in with the Identity Provider, but a successful login event does not show up in Auth0 logs`.

Also, bear in mind that, according to design, we should minimise our use of italics. This is on my todo list for June, but in the meanwhile try to minimise it's usage, especially in titles.